server/src/workspace/workspace-resolver-builder/interfaces/workspace-resolvers-builder.interface.ts

```diff
@@ -39,10 +39,22 @@ export interface UpdateOneResolverArgs<Data extends Record = Record> {
   data: Data;
 }
 
+export interface UpdateManyResolverArgs<
+  Data extends Record = Record,
+  Filter = any,
+> {
+  filter: Filter;
+  data: Data;
```

Comment:
It's an update many, so `Data` should be an array:

```suggestion
  data: Data[];
```

Reply:
@magrinj I don't think this should be an array, we're using the `update` method of pgraphql which expects a `set` and `filter` param, we're passing in just one value for `set` which is an object with updates, not an array of objects unlike the `insertInto[tableName]Collection` method which accepts an objects array. There's no updateMany method.